src/call_builder.js

```diff
@@ -174,28 +176,24 @@ export class CallBuilder {
   _parseResponse(json) {
     if (json._embedded && json._embedded.records) {
       return this._toCollectionPage(json);
-    } else {
-      return this._parseRecord(json);
     }
+    return this._parseRecord(json);
   }
 
   /**
    * @private
    */
-  _toCollectionPage(json) {
-    for (var i = 0; i < json._embedded.records.length; i++) {
+  _toCollectionPage(jsonParam) {
+    const json = jsonParam;
```

Comment:
We can't we use `json` as a name of a function parameter?

Reply:
This is antipattern (function parameters reassigning) and there is a [https://eslint.org/docs/rules/no-param-reassign](rule) in `eslint`.

Reply:
That would be if you reassigned the actual parameter:

```
function(json) {
   json = someNewJson;
}
```

The changed code right now just obfuscates that you're actually updating the passed in object. If it was truly meant to have no side effects it would need to be cloned before assigning to it.